ffi/generate_cgo.go

```diff
@@ -51,27 +53,20 @@ func main() {
 	fmt.Printf("Successfully switched CGO directives to %s mode\n", mode)
 }
 
-// Get the FIREWOOD_LD_MODE environment variable. If it's not set, look for local libs,
-// and if found, set it to "LOCAL_LIBS". Otherwise, set it to "STATIC_LIBS".
-func localOrStaticLibs() string {
+// getFirewoodLdMode returns the FIREWOOD_LD_MODE environment variable.
+// Defaults to "LOCAL_LIBS".
+func getFirewoodLdMode() string {
 	mode, ok := os.LookupEnv("FIREWOOD_LD_MODE")
```

Comment:
nit: we should probably check that `FIREWOOD_LD_MODE` is one of the two possible values, I can imagine if there's a typo `FIREWOOD_LDMODE=STATIC_LIBS` while having the real mc-coy in your environment set to the other value confusing someone.